### PR TITLE
feat(covid19): add covid-19 page

### PR DESCRIPTION
**Pull request by linooohon**
## Types of Changes

**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies.

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

- Add the new version of the COVID-19 guidelines page.
- Closed #45 which is the old version.

## Step to test this PR

The button to go to the COVID-19 guidelines page is next to the sign-in button in the navbar.

### CODE REVIEW

File: i18n/covid-19/index.i18n.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            '休閒娛樂場所例如電影院、運動中心、圖書館、藝文展演、酒吧、八大場所全部關閉。',
+            '停止室內5人、室外10人以上的社交聚會、家庭聚會。',
+            '餐飲場所必須實聯制、使用隔板，否則只能外帶或外送。',
+            '二三級警戒區域之間，避免不必要的移動。',
+        ],
+        governmentDocumentsTitle: '政府機關公文',
+        governmentDocumentsContent: [
+            {
+                title: '衛生福利部疾病管制署-嚴重急性呼吸道症候群防治工作手冊',
+                link: 'https://www.cdc.gov.tw/File/Get/s8bfTZsDHo4V2CE2-ozrDg',
```
Comment (mattwang44) [inline i18n/covid-19/index.i18n.js:70]: The link is wrong. Let me contact venue team first.

File: i18n/covid-19/index.i18n.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            },
+        ],
```
Comment (mattwang44) [inline i18n/covid-19/index.i18n.js:72]: Same content in zh & en version. Please put the content in the `.vue`.

File: i18n/covid-19/index.i18n.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        licenseTitle: '授權 (CC BY-SA 3.0 TW)',
+        licenseContent: '此文件採用 {license} 授權條款。',
+        licensePic: 'https://i.imgur.com/cVjN760.jpg=200x',
```
Comment (mattwang44) [inline i18n/covid-19/index.i18n.js:75]: Same picture in zh & en version. Please move this out from i18n.js.

File: pages/covid-19/guidelines.vue [hunk 4]
```diff
@@ -1,11 +1,94 @@
...
-    <p>COVID 19 Guidelines</p>
+    <i18n-page-wrapper class="covid19 px-10 md:px-20 lg:px-36 xl:px-60">
+        <core-h1 :title="$t('pageTitle')" class="covid19__pageTitle"></core-h1>
+        <div class="covid19__summary">
+            <p
+                v-for="(item, index) in $t('latestEditedTime')"
+                :key="`latestEditedTime_${index}`"
+            >
+                {{ item }}
+            </p>
```
Comment (mattwang44) [inline pages/covid-19/guidelines.vue:10]: - Replace `flex justify-center` with [`text-right`](https://tailwindcss.com/docs/text-align) for this section.
- Translate only for those words that needed.
```suggestion
            <p>
                (CC BY-SA 3.0 TW)
                {{ $t('XXXX') }}: 2021-06-15
            </p>
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]

File: pages/covid-19/guidelines.vue [hunk 5]
```diff
@@ -1,11 +1,94 @@
...
+            </ol>
+        </div>
+        <div class="covid19__license">
+            <h3 path="licenseTitle">
+                {{ $t('licenseTitle') }}
+            </h3>
+            <i18n tag="p" path="licenseContent">
+                <template #license>
+                    <ext-link
+                        href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW"
```
[the remark this answers is not in the document]
Comment (linooohon) [reply pages/covid-19/guidelines.vue]: Ok, I got it!

File: pages/covid-19/guidelines.vue [hunk 6]
```diff
@@ -1,11 +1,94 @@
...
+                        item.title
+                    }}</ext-link>
+                </ul>
+            </ol>
+        </div>
+        <div class="covid19__license">
+            <h3 path="licenseTitle">
+                {{ $t('licenseTitle') }}
+            </h3>
+            <i18n tag="p" path="licenseContent">
```
[the remark this answers is not in the document]
Comment (linooohon) [reply pages/covid-19/guidelines.vue:46]: Sorry, I missed this one.

File: i18n/covid-19/index.i18n.js (hunk 1 above)
Comment (linooohon) [reply i18n/covid-19/index.i18n.js]: Okay, thank you!

**Comment by linooohon** [comment]
In this commit, I fixed i18n, style, and structure. [751e81d](https://github.com/pycontw/pycontw-2021/pull/91/commits/751e81ddb3de1c178cc2db137636195cf5f82a08)
@mattwang44 And thanks for your review!
So, now just wait for the link of government documents to be changed.
Thanks!

File: pages/covid-19/guidelines.vue [hunk 7]
```diff
@@ -76,19 +71,49 @@ export default {
...
+            },
+            licensePic: 'https://i.imgur.com/cVjN760.jpg=200x',
+        }
+    },
 }
 </script>
 <style lang="postcss" scoped>
 .covid19 h3 {
-    @apply text-center pt-5 pb-5;
+    @apply text-center pt-5 pb-5 text-base;
```
Comment (mattwang44) [inline pages/covid-19/guidelines.vue:87]:
```suggestion
    @apply text-center pt-5 pb-5 text-base md:text-xl;
```

File: pages/covid-19/guidelines.vue [hunk 8]
```diff
@@ -76,19 +71,49 @@ export default {
...
     color: #9387ff;
 }
 
-.covid19__summary p {
-    @apply flex justify-center;
+.covid19 p,
+.covid19 ol,
+.covid19 a {
+    @apply text-sm;
```
Comment (mattwang44) [inline pages/covid-19/guidelines.vue:94]:
```suggestion
    @apply text-sm md:text-base;
````

File: pages/covid-19/guidelines.vue [hunk 9]
```diff
@@ -76,19 +71,49 @@ export default {
...
 }
 
 .covid19__guidelines ol {
     @apply list-decimal;
 }
+
+.covid19__guidelines ol li {
+    @apply text-xs;
```
Comment (mattwang44) [inline pages/covid-19/guidelines.vue:102]:
```suggestion
    @apply text-xs md:text-base;
```

File: pages/covid-19/guidelines.vue [hunk 10]
```diff
@@ -76,19 +71,49 @@ export default {
...
+
+    .covid19 p,
+    .covid19 ol,
+    .covid19 a {
+        @apply text-base;
+    }
+    .covid19__guidelines ol li {
+        @apply text-base;
+    }
+}
```
Comment (mattwang44) [inline pages/covid-19/guidelines.vue:118]:
```suggestion
```

**Review by mattwang44 — COMMENTED** [review]
use `md:` instead of media query here is recommended
https://tailwindcss.com/docs/breakpoints

**Comment by linooohon** [comment]
#98 the updated commit is here.

**Pull request closed without merging**